resolwe/flow/executors/startup_processing_container.py

```diff
@@ -344,8 +344,13 @@ def __init__(self, loop):
         self.exported_files_mapper = dict()  # type: Dict[str, str] # noqa: F821
         self.upload_socket = None
         self.update_logs_future = None
-        self._uploading_log_files_lock = asyncio.Lock(loop=loop)
-        self._send_file_descriptors_lock = asyncio.Lock(loop=loop)
+        # Drop this when we stop supporting Python 3.7.
+        # This class has been implicitly getting the current running loop since 3.7.
+        lock_arguments = {}
+        if sys.version_info < (3, 7):
```

Comment:
The comment says 3.7, but here it is 3.6.

Reply:
The comment is wrong, fixed.